java/vector/src/test/java/org/apache/arrow/vector/TestListVector.java

```diff
@@ -811,14 +811,9 @@ public void testSetInitialCapacity() {
       assertEquals(512, vector.getValueCapacity());
       assertEquals(8, vector.getDataVector().getValueCapacity());
 
-      boolean error = false;
-      try {
-        vector.setInitialCapacity(5, 0.1);
-      } catch (IllegalArgumentException e) {
-        error = true;
-      } finally {
-        assertTrue(error);
-      }
+      vector.setInitialCapacity(5, 0.1);
+      vector.allocateNew();
+      assertEquals(7, vector.getValueCapacity());
```

Comment:
how about adding `assertEquals(1, vector.getDataVector().getValueCapacity())` and also maybe a brief explanation of the values?

Reply:
Done.